README.rst

```diff
@@ -616,6 +616,28 @@ will order the results by the distance to the point (-122.4862, 37.7694).
 We can also reverse the order of the results by passing ``order=desc``:
 ``/location/?point=-122.4862,37.7694&order=desc&format=json``
 
+Schema Generation
+=================
```

Comment:
```suggestion
-----------------
```

Let's keep consistency with the heading levels used. I also noticed the headings below this point are messed (bigger than the preceding ones), will fix that later.